common-custom-user-data-gradle-plugin/README.md

```diff
@@ -2,17 +2,39 @@
+1. Apply the published plugin directly in your build and immediately benefit from enhanced Build Scans
+2. Copy this repository and develop a customized version of the plugin which will standardize Gradle Enterprise usage across multiple projects
 
-In order for the Common Custom User Gradle plugin to become active, you need to apply it in the `settings.gradle[.kts]` file of your project. The `settings.gradle[.kts]` file is the same file where you have already declared the Gradle Enterprise Gradle plugin. See [here](https://github.com/gradle/gradle-enterprise-build-config-samples/blob/master/common-custom-user-data-gradle-plugin/settings.gradle) for an example.
+Out of the box, this plugin will enhance all Build Scans published with additional tags, links and custom values. These include:
+- A tag representing the Operating System.
+- A tag representing how the build was invoked, be that from your IDE (IDEA, Eclipse, Android Studio) or from the command-line.
+- A `CI` tag for builds run on CI. In addition, a set of tags, links and custom values specific to the CI server will be added.
+  - The following CI servers are supported: Jenkins, Team City, Circle CI, Bamboo, GitHub Actions, Gitlab and Travis.
+- For `git` repositories, custom values with the Commit ID, Branch and the output of `git status`
 
-If you are still using a 5.x version of Gradle, apply the Common Custom User Gradle plugin to the `build.gradle[.kts]` file of your root project.
+### Applying the published plugin
+
+The Common Custom User Data Gradle Plugin is available in the [Gradle Plugin Portal](https://plugins.gradle.org/plugin/com.gradle.common-custom-user-data-gradle-plugin). This plugin requires the the [Gradle Enterprise Gradle plugin](https://plugins.gradle.org/plugin/com.gradle.enterprise) also be applied in your build in order to have an effect.
+
+To benefit from the Common Custom User Gradle plugin in your project, you must apply it in the `settings.gradle[.kts]` file of your project. This is the same file where you have already declared the Gradle Enterprise Gradle plugin. See [here](https://github.com/gradle/gradle-enterprise-build-config-samples/blob/master/common-custom-user-data-gradle-plugin/settings.gradle) for an example.
+
+If you are still using a 5.x version of Gradle, apply the plugin to the `build.gradle[.kts]` file of your root project.
+
+### Developing a customized version of the plugin
+
+For more power and flexibility, we recommend creating a copy of this repository so that you may develop a customized version of the plugin and publish it locally for your projects to consume.
+
+This approach has a number of benefits:
+- Add, remove and change the set of tags, links and custom values added to each Build Scan. This allows you to customize your Build Scans for your particular needs.
+- Standardize Gradle Enterprise and Gradle Build Cache configuration in your organization, removing the need for each project to specify this configuration.
```

Comment:
Not sure what you mean by the `Standardize Gradle Enterprise` part. Is it the URL/allowUntrusted configuration?